.github/workflows/pr_env.yaml

```diff
@@ -110,13 +110,15 @@ jobs:
             wanda_postgres_user='${{ env.PR_NUMBER}}wanda' \
             web_container_name='${{ env.PR_NUMBER}}web' \
             web_container_image='ghcr.io/trento-project/trento-web:${{ env.PR_NUMBER}}-env' \
+            wanda_container_image='ghcr.io/trento-project/trento-wanda:demo' \
             wanda_container_name='${{ env.PR_NUMBER}}wanda' \
             docker_network_name='${{ env.PR_NUMBER}}net' \
             nginx_vhost_filename='${{ env.PR_NUMBER}}' \
             trento_server_name='${{ env.PR_BASE_URL}}' \
             web_postgres_password='trento' \
             wanda_postgres_password='wanda' \
             rabbitmq_vhost='${{ env.PR_NUMBER}}' \
+            rabbitmq_username='rabbitusr_${{ env.PR_NUMBER}}' \
```

Comment:
Could we maybe simply use `env.PR_NUMBER` or at least revert the order `${{ env.PR_NUMBER}}rabbitusr` as all the variables follow this pattern?